src/ProductConstructionService/ProductConstructionService.DependencyFlow/PullRequestUpdater.cs

```diff
@@ -1236,11 +1236,11 @@ await UpdateCodeFlowPullRequestAsync(
                     subscription,
                     codeFlowRes.ConflictedFiles,
                     build),
-                CommentType.Warning);
```

Comment:
since these are created only in the `PullRequestUpdater` we probably don't have to make darc handle them